lib/rubyblok/mixins/webhook.rb

```diff
@@ -25,6 +29,26 @@ def create
         def model
           Rubyblok.configuration.model_name.classify.constantize
         end
+
+        def update_story(payload)
+          storyblok_story = Rubyblok::Services::GetStoryblokStory.call(slug: payload['story_id'])
+          storyblok_story = Rubyblok::Services::ReplaceStoryblokUrl.call(story: storyblok_story)
+          model.find_or_create(storyblok_story)
+        end
+
+        def verify_signature(payload_body)
+          signature =
+            OpenSSL::HMAC.hexdigest(OpenSSL::Digest.new('sha1'), webhook_secret, payload_body)
+          raise StandardError, 'Signature check failed!' unless verified?(signature)
+        end
+
+        def verified?(signature)
+          Rack::Utils.secure_compare(signature, request.env['HTTP_WEBHOOK_SIGNATURE'])
+        end
+
+        def webhook_secret
+          Rubyblok.configuration.webhook_secret
+        end
```

Comment:
```suggestion
        def webhook_secret
          @webhook_secret ||= Rubyblok.configuration.webhook_secret
        end
```

Reply:
yes, it makes sense, however it is not an expensive operation, so it would not make too much improvement